pkg/provider/resource_cluster.go

```diff
@@ -361,8 +361,8 @@ func (c *ClusterResource) retryFunc(ctx context.Context, d *schema.ResourceData,
 			return resource.NonRetryableError(fmt.Errorf("Error describing instance: %s", err))
 		}
 
-		if !client.HasOkCondition(cluster.Conditions) {
-			return resource.RetryableError(errors.New("Instance not yet ready"))
+		if !cluster.IsHealthy() {
+			return resource.RetryableError(errors.New("Instance not yet health"))
```

Comment:
```suggestion
			return resource.RetryableError(errors.New("Instance not yet ready"))
```
I'd reject this change and keep it with `Instance not yet ready` for now.